packages/mobile/src/screens/edit-track-screen/fields/PriceAndAudienceField/PremiumRadioField/BoxedTextField.tsx

```diff
@@ -25,21 +27,24 @@ const useStyles = makeStyles(({ spacing, palette }) => ({
 type BoxedTextFieldProps = TextFieldProps & {
   title: string
   description: string
+  textField?: ReactNode
 }
 
 export const BoxedTextField = (props: BoxedTextFieldProps) => {
-  const { title, description, style, ...other } = props
+  const { title, description, textField, style, ...other } = props
   const styles = useStyles()
 
   return (
     <View style={[styles.root, style]}>
       <Text weight='bold'>{title}</Text>
       <Text>{description}</Text>
-      <TextField
-        style={styles.textField}
-        styles={{ root: styles.textInput }}
-        {...other}
-      />
+      {textField ?? (
```

Comment:
this confuses me. We're passing in a node but if truthy we're not rendering it, and instead rendering our own text field?

should this be:
```
// textField: TextFieldProp

TextFieldProp
	? <TextFieldProp {...other} />
	: <TextField {...other} />

```

Reply:
great spot!! this was a holdover from when i wanted to make it a ReactNode, then realized it should be an element, but didnt remove this defininition